product_docs/docs/biganimal/release/administering_cluster/04_customizing_policy_ definitions.mdx

```diff
@@ -36,9 +37,11 @@ BigAnimal does not customize your Azure policies to prevent conflicts with exter
 
 You're now ready to monitor, identify, and remediate non-compliant resources to improve the compliance state of the resources in your subscription.
 
-### Customizable policy definition parameters
+#### Customizable policy definition parameters
+
+While customizing Azure policies, refer to the following list of recommended parameters and values that are based on BigAnimal's resource configurations. 
 
-You can customize the following parameters in your baseline Azure policies. Use the values below each parameter while configuring the default ASC policy of a subscription. 
+Use the values below each parameter while configuring the default ASC policy of a subscription. 
```

Comment:
Is "ASC" still accurate here? They've renamed the product, right?

Reply:
Yeah, but strangely they have stuck with the ASC nomenclature in the default policy assignments.